x-pack/plugins/maps/public/classes/layers/vector_layer/utils.tsx

```diff
@@ -69,13 +69,15 @@ export async function syncVectorSource({
   } = syncContext;
   const dataRequestId = SOURCE_DATA_REQUEST_ID;
   const requestToken = Symbol(`${layerId}-${dataRequestId}`);
-  const canSkipFetch = await canSkipSourceUpdate({
-    source,
-    prevDataRequest,
-    nextMeta: requestMeta,
-    extentAware: source.isFilterByMapBounds(),
-  });
-  if (canSkipFetch) {
+  const doRefresh = syncContext.forceRefresh
```

Comment:
how about keeping this as canSkipFetch so its more similiar to other sync flows.

```
const canSkipFetch = syncContext.forceRefresh
    ? false
    : await canSkipSourceUpdate({
        source,
        prevDataRequest,
        nextMeta: requestMeta,
        extentAware: source.isFilterByMapBounds(),
      });
```